packages/node-app/src/swagger.json

```diff
@@ -536,6 +536,86 @@
                     }
                 }
             }
+        },
+        "/login": {
+            "post": {
+                "tags": ["authentication"],
+                "description": "Login user",
+                "parameters": [
+                    {
+                      "name": "tokenType",
+                      "in": "query",
+                      "required": true
+                    },
+                    {
+                        "name": "accessToken",
+                        "in": "query",
+                        "required": true
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "User",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "example": {
+                                        "token": "auth-token"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "401": {
+                        "description": "Invalid Authentication"
+                    },
+                    "5XX": {
+                        "description": "Unexpected error"
+                    }
+                }
+
+            }
+        },
+        "/me": {
+            "get": {
+                "tags": ["authentication"],
+                "description": "Get user",
+                "responses": {
+                    "201": {
```

Comment:
This should be 200 I believe

Reply:
No, the endpoint is returning a 201

Reply:
This is relative to /me endpoint. This returns a 200 with the user object